utils/ssh.py

```diff
@@ -262,15 +302,16 @@ def cpu_spike(self, seconds=60, cpus=2, **kwargs):
             "for ((i=0; i<instances; i++)) do while (($(date +%s) < $endtime)); "
             "do :; done & done".format(seconds, cpus), **kwargs)
 
-    def run_rails_command(self, command, timeout=RUNCMD_TIMEOUT):
+    def run_rails_command(self, command, timeout=RUNCMD_TIMEOUT, **kwargs):
         logger.info("Running rails command `%s`", command)
-        return self.run_command('cd /var/www/miq/vmdb; bin/rails runner {}'.format(command),
-            timeout=timeout)
+        return self.run_command('/var/www/miq/vmdb/bin/rails runner {}'.format(command),
```

Comment:
@mshriver It should not affect, as we are invoking rails directly. why I removed "cd" is because it does not work with "sudo". 

@psav are you OK with this change?